src/main/java/com/faforever/client/fa/relay/ice/IceAdapterImpl.java

```diff
@@ -154,10 +155,16 @@ public void onGpgGameMessage(GpgOutboundMessageEvent event) {
 
       Path workDirectory = Path.of(nativeDir).toAbsolutePath();
 
+      var classpath = getBinaryName(workDirectory);
```

Comment:
We don't really use `var` anywhere else so please use the full type 😝

Reply:
Ok... I did use it a lot lately. And I like it. Especially now Intellij always shows the type of vars

Reply:
Yeah understood although I don't know how I feel about it yet